mobile/.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "@callstack/eslint-config/node"
+  }
```

Comment:
In order to use Callstack's eslint config in the React Native project you can just specify it as: `@callstack`.

For more information please refer to this [site](https://github.com/callstack/eslint-config-callstack?tab=readme-ov-file#usage).

Reply:
Sure, will keep it in mind going ahead
Thanks for reviewing !